influxdb/influxdb.go

```diff
@@ -0,0 +1,76 @@
+package influxdb
+
+import (
+	"strconv"
+	"time"
+
+	client "github.com/influxdata/influxdb/client/v2"
+	"github.com/mainflux/mainflux"
+)
+
+var _ Writer = (*influxRepo)(nil)
+
+type influxRepo struct {
+	database  string
+	pointName string
+	client    client.Client
+}
+
+type fields map[string]interface{}
+type tags map[string]string
+
+func convertMsg(msg mainflux.Message) (tags, fields) {
+	time := strconv.FormatFloat(msg.Time, 'f', -1, 64)
+	update := strconv.FormatFloat(msg.UpdateTime, 'f', -1, 64)
+	tags := map[string]string{
```

Comment:
Can you please explain this design - why all these fields are tags? Tags are indexed and time-graphs can be made for one tag, so I think that the `Channel` should be the only tag here.

Reply:
Since data tags are used to record metadata, it made sense for me to put all `non-value` data to tags, and value data to `fields`.

Reply:
This is not the way InfluxDB works. You need to go through tutorials and understand well the meaning of tags and fields, it is crucial.

This will also have impact on Grafana integration for visualization of messages coming over certain channel.